esp/scm/ws_topology.ecm

```diff
@@ -59,7 +59,17 @@ ESPStruct TpCluster
     [min_ver("1.15")] bool   HasThorSpareProcess;
     [min_ver("1.12")] ESParray<ESPstruct TpMachine> TpMachines;
 };
+
 //  ===========================================================================
+ESPStruct TpServiceTarget
+{
+    string Name;
+    string Type;
+    unsigned Port;
+    bool TLSSecure;
+    bool Public;
+    string Target;
```

Comment:
what is target? not in any of the service type schema's/or in values.yaml afaics.

Reply:
I saw the target from the roxie service:
services:
      - name: roxie
        type: roxie
        port: 9876
        target: roxie

Reply:
right ok, I see. The target is the roxie, the service name of this roxie is also named roxie.